### PR TITLE
Add python tutorial

### PR DESCRIPTION
**Pull request by Deniallugo**

### CODE REVIEW

File: docs/api/sdk/python/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+## Installation
+
+ZkSyncSDK can be installed (preferably in a virtualenv) using pip as follows:
+
+```
+$ pip install git+https://github.com/zksync-sdk/zksync-python.git 
+```
+
+Unfortunately, currently, SDK is not published on <pip>, thus install the dependency through the repository is
```
Comment (ly0va) [inline docs/api/sdk/python/tutorial.md]: The angle brackets make words look like HTML tags to markdown, so they are not displayed
```suggestion
Unfortunately, currently, SDK is not published on PyPI, thus installation through the git repository is
```

File: docs/api/sdk/python/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+For using this library:
+ 1. You have to download zksync-crypto-library from <https://github.com/zksync-sdk/zksync-crypto-c/releases>
+ 2. Set env variable `ZK_SYNC_LIBRARY_PATH` with a path to the downloaded library 
+
+## Initialize crypto library
+```
+from zksync_sdk import ZkSyncLibrary
+
+lib = ZkSyncLibrary()
+```
```
Comment (ly0va) [inline docs/api/sdk/python/tutorial.md]: Probably should mark all code blocks as `python`, for proper syntax highlighting

File: docs/api/sdk/python/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+# Verified state is final
+verifiedETHBalance = await wallet.get_balance("ETH", "verified")
+```
+
+To list all tokens of this account at once, use `account_info`:
+
+```python
+account_state = await wallet.get_account_state()
+committed_eth_balance = account_state.committed.balances.get("ETH")
+verified_dai_balance = account_state.balance.balances.get("DAI")
```
Comment (ly0va) [inline docs/api/sdk/python/tutorial.md]:
```suggestion
committed_eth_balance = account_state.committed.balances.get("ETH")
verified_dai_balance = account_state.verified.balances.get("DAI")
```

File: docs/api/sdk/python/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+## Making a transfer in zkSync
+
+For making transfer to another account you just need to set receiver amount and token 
+```python
+tr = await wallet.transfer("0x21dDF51966f2A66D03998B0956fe59da1b3a179F",
+                           amount=Decimal("0.01"), token="USDC")
+
+```
+For better controlling, you can optionally provide `nonce` and `fee`
```
Comment (ly0va) [inline docs/api/sdk/python/tutorial.md]:
```suggestion
If you want more control over the transaction, you can optionally provide `nonce` and `fee`
```

**Review by StanislavBreadless — COMMENTED** [review]
When I run the zksync-docs on my localhost, all the SDKs are shown correctly, but this one is not. 

I could not identify the problem :(

**Comment by Deniallugo** [comment]
> When I run the zksync-docs on my localhost, all the SDKs are shown correctly, but this one is not.
> 
> I could not identify the problem :(

I've added the necessary changes and now everything works

**Review by StanislavBreadless — COMMENTED** [review]
Please add a section about how to get tx information based on hash (the clients find it important) and we are good to go

**Review by StanislavBreadless — APPROVED** [review]
LGTM

**Pull request merged**